### PR TITLE
Minion Cert Manager POC

### PR DESCRIPTION
**Pull request by geraldhumphries**
## Description

Draft PR to hand off minion cert manager work to @BenjaminJ 

## Jira link(s)
- https://issues.opennms.org/browse/AR-44

## Checklist
* [ ] Follows Horizon Stream's [development guidelines.](https://github.com/OpenNMS/horizon-stream/wiki/Development-Guidelines)
* [ ] Appropriate reviewer(s) have been selected.
* [ ] Jira issue(s) have been updated to "In Review".
* [ ] Includes [appropriate tests.](https://github.com/OpenNMS/horizon-stream/wiki/Test-Strategy)
* [ ] Documentation has been updated as necessary.
* [ ] Notify devops of changes to the Charts

### CODE REVIEW

File: rest-server/src/main/resources/application.yml [hunk 1]
```diff
@@ -31,5 +33,6 @@ grpc:
     notification: localhost:15065
     alerts: opennms-alert:6565
     flows: flows.onms-dp-dev.dev.nonprod.dataservice.opennms.com:443 # temp for development
+    minionCertificateManager: minion-certificate-manager:8990
```
Comment (geraldhumphries) [inline rest-server/src/main/resources/application.yml:36]: @BenjaminJ This should be overridden by an environment variable here, but for some reason it was not working: https://github.com/OpenNMS-Cloud/horizon-stream/pull/961/files#diff-7d9cd61e2e13bc14fdcfa1921c0949d8c36a25751459ca60ee93d25bc3eb355bR65

Setting this value to `opennms-minion-certificate-manager:8990` will match what's in K8s and get the requests from the BFF working, but the environment variable should definitely be used instead.

File: shared-lib/minion-certificate-manager-api/src/main/proto/minion-certificate-manager.proto [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+message GetMinionCertificateRequest {
+  string tenant_id = 1;
+  uint64 location_id = 2;
+}
+
+message GetMinionCertificateResponse {
+  string tenant_id = 1;
+  uint64 location_id = 2;
+  bytes zip = 3;
```
[the remark this answers is not in the document]
Comment (BenjaminJ) [reply shared-lib/minion-certificate-manager-api/src/main/proto/minion-certificate-manager.proto:46]: Changes according to the confluence page would be:

```
message GetMinionCertificateResponse {
  bytes zip = 1;
  string encryptionKey = 2;
}
```

@kbakanov is that works for you ?

**Comment by geraldhumphries** [comment]
@BenjaminJ looks like #996 supersedes this, can I close this PR?

**Comment by BenjaminJ** [comment]
@geraldhumphries yes we can

**Pull request closed without merging**